polkadot-protocol/onchain-governance/overview.md

```diff
@@ -44,7 +44,7 @@ In OpenGov, origins and tracks are central to managing proposals and votes.
 
 Developers must be aware that referendums from different origins and tracks will take varying amounts of time to reach approval and enactment. The [Polkadot Technical Fellowship](https://wiki.polkadot.network/docs/learn-polkadot-technical-fellowship){target=\_blank} has the option to shorten this timeline by whitelisting a proposal and allowing it to be enacted through the [Whitelist Caller](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#whitelisted-caller){target=\_blank} origin.
 
-Visit [Origins and Tracks Info](https://wiki.polkadot.network/docs/learn-polkadot-opengov-origins#origins-and-tracks-info){target=\_blank} for details on current origins and tracks, associated terminology, and parameters.
+Visit [Origins and Tracks Info](/){target=\_blank} for details on current origins and tracks, associated terminology, and parameters.
```

Comment:
@eshaben Sorry, I'm struggling to understand this change... what would be the reason for "(/)" here?

Reply:
Mehhhh I must have somehow deleted it. It was supposed to change to be /polkadot-protocol/onchain-governance/origins-tracks/

Reply:
I figured we should link to our page internally, but if we're not going to document the parameters and such, maybe we need two separate links here. One to link to our docs and one for the wiki

Reply:
yep @eshaben @nhussein11 this should be Wiki page - our page is high level overview, slimmed down without all the info about paramiters